docs/blog/2025-01-28-daily-uploads-and-two-year-review/index.html

```diff
@@ -168,7 +168,7 @@ <h2 id="what's-next%3F" tabindex="-1">What's Next?</h2>
       </div>
     </div>
     <hr>
-    <div class="content m-10 has-text-centered m-5"><a href="/blog/2024-05-14-rum-insights-baseline/">&larr; RUM Archive Insights with Baseline Support</a>
+    <div class="content m-10 has-text-centered m-5"><a href="/blog/2024-05-14-rum-insights-baseline/">&larr; RUM Archive Insights with Baseline Support</a><a href="/blog/2025-08-akamai-employee-individual-datasets/">Akamai Employee Individual Datasets &rarr;</a>
```

Comment:
It seems like there is no whitespace between `...Baseline Support` and `Akamai Employee...`

Reply:
It's unclear to me what triggered this change since the corresponding `.md` file hasn't changed.

Reply:
This is just the Left / Right navigation links at the bottom.

But they look squished

<img width="1628" height="180" alt="image" src="https://github.com/user-attachments/assets/5e436da9-4e17-4215-921d-b1750cbb531d" />

I've fixed it:

<img width="1652" height="160" alt="image" src="https://github.com/user-attachments/assets/20f80987-1cae-4ef8-9198-584140815970" />